ddtrace/utils/uwsgi.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from ddtrace import _worker
+from ..internal.logger import get_logger
+
+logging.basicConfig(level=logging.DEBUG)
+
+log = get_logger(__name__)
+
+
+def check_threads_enabled():
```

Comment:
Should we have this function return `True`/`False`?

Just thinking, if someone doesn't have debug logging enabled they won't see the messages, which is probably ok since when we use this function somewhere else we might raise an exception, or... who knows what we'll do.

Reply:
I will change the function to return true if uWSGI is running and the `enable-threads` config is true.  False if uWSGI import fails since other scenarios are covered by the debug logs.